static/js/partners.js

```diff
@@ -15,7 +15,7 @@ var partners = {}
 /**
  * Loops through the partner clouds and loads the feeds for each.
  */
-partners.loadPartnerClouds = function() {
+partners.loadPartnerClouds = function(partnerLogoClouds) {
```

Comment:
I would rather this was:

``` javascript
partners.loadPartnerCloud = function (queryParams, elementId) {
```

But if you want to merge this for now I can tidy it up afterwards.